docs/administration/5_troubleshooting.md

```diff
@@ -41,15 +41,22 @@ Common reasons that it may not pass include:
 
 Correct any errors if shown and re-run the federation tester to check the results.
 
-## 3. System time
+## 3. STUN/TURN Server tester
```

Comment:
STUN/TURN troubleshooting should probably be at the end of the list, I don't think it's a "common" issue. System time or file descriptors are more likely.

Reply:
Noted and changed to place STUN/TURN Server tester to the bottom